keychain-environment.el

```diff
@@ -49,6 +49,29 @@
 
 ;;; Code:
 
+(defvar keychain-should-inherit
+  nil
+  "Should the keychain invocation should inherit.
+Can be either nil, local, any or t. See keychain manual.")
+
+(defvar keychain-be-quick
+  t
+  "Should keychain use the -Q option. I think there is a bug in
+keychain somewhere that means ssh-agent ends up being spawned anyway if
+  -Q is no specified")
+
+(defun keychain-command (type)
+  "Return keychain invocation for `TYPE'."
+  (format "keychain %s -q --noask %s --eval"
+          (if keychain-be-quick
+              "-Q"
+            "")
+          (case keychain-should-inherit
+            ('nil (format "--noinherit --agents %s" type))
+            ('local "--inherit local")
+            ('any "--inherit any")
+            ('t "--inherit local-once"))))
```

Comment:
I don't think those are supposed to be quoted. Maybe that works for `case` but not `cl-case`. Maybe it works for the latter too, but I have never seen this before.